projects/pgai/pgai/semantic_catalog/templates/table.j2

```diff
@@ -12,4 +12,5 @@ ALTER TABLE {{ table.schema_name }}.{{ table.table_name }} ADD CONSTRAINT {{ con
 {% for column in (table.columns if table.columns is not none else []) -%}
 {% if column.description is not none %}COMMENT ON COLUMN {{ table.schema_name }}.{{ table.table_name }}{{ column.name }} IS $${{ column.description.description }}$$;{% endif -%}
 {% endfor -%}
+{% if table.sample is not none %}{{ table.sample }}{% endif -%}
```

Comment:
should we have <samples> or anything here?

Reply:
Not sure I follow. What would we put if the sample is missing?

Reply:
sorry should we have `<samples></samples>` here